frontend/components/Company.vue

```diff
@@ -0,0 +1,205 @@
+            overflow-hidden
+            border-b
+            rounded-xl
+            border-gray-200
+            sm:rounded-lg
+            w-full
+          "
+        >
+          <table class="min-w-full divide-y divide-gray-200">
+            <thead style="backgound: #e5e5e5">
+              <tr>
+                <th
+                  scope="col"
+                  class="
+                    px-6
+                    py-3
+                    text-left text-xs
+                    font-bold
+                    text-gray-500
+                    uppercase
+                    tracking-wider
+                  "
+                  style="color: #b1b1b1; font-family: 'Inter', sans-serif"
+                >
+                  Position
+                </th>
+                <th
+                  scope="col"
+                  class="
+                    px-6
+                    py-3
+                    text-left text-xs
+                    font-medium
+                    text-gray-500
+                    uppercase
+                    tracking-wider
+                  "
+                  style="color: #b1b1b1; font-family: 'Inter', sans-serif"
+                >
+                  Company Name
+                </th>
+                <th
+                  scope="col"
+                  class="
+                    px-6
+                    py-3
+                    text-left text-xs
+                    font-bold
+                    text-gray-500
+                    uppercase
+                    tracking-wider
+                  "
+                  style="color: #b1b1b1; font-family: 'Inter', sans-serif"
+                >
+                  Jobs title
+                </th>
+                <th
+                  scope="col"
+                  class="
+                    px-6
+                    py-3
+                    text-left text-xs
+                    font-bold
+                    text-gray-500
+                    uppercase
+                    tracking-wider
+                  "
+                  style="color: #b1b1b1; font-family: 'Inter', sans-serif"
+                >
+                  Salary
+                </th>
+                <th
+                  scope="col"
+                  class="
+                    px-6
+                    py-3
+                    text-left text-xs
+                    font-bold
+                    text-gray-500
+                    uppercase
+                    tracking-wider
+                  "
+                  style="color: #b1b1b1; font-family: 'Inter', sans-serif"
+                >
+                  Rating
+                </th>
+                <th scope="col" class="relative px-6 py-3">
+                  <span class="sr-only">Edit</span>
+                </th>
+              </tr>
+            </thead>
+            <tbody
+              class="bg-white divide-y divide-white"
+              v-for="company in companies.slice(0, 4)"
+              :key="company"
```

Comment:
Hi Assam you did a great job but i have some questions

why are you using the whole object hier "company" as key?? 
is there no id attribut in the object??
i think i will be better to use it so do somthing like :key="company.id". 
if there is no id you can also create someone when looping. yo can do v-for="(compay, index) in companies.slice(0, 4)" and then put index as key.

Reply:
Oup's ! Indeed, the best would be to use an id of the object and which besides already exists. Thank you